code/vocabulary.py

```diff
@@ -1,9 +1,66 @@
 from talon import Context, Module, actions, grammar
 from .user_settings import bind_list_to_csv, bind_word_map_to_csv
 
+mod = Module()
```

Comment:
Reordering this file makes it harder to verify your claim that it doesnt change user.text, which is made extra difficult by our lack of ability to write unit tests. I do tend to prefer the reorganization that has been done, just I'd rather smaller, easier to manually verify changes.

Reply:
Good point, in future I will try to make the changes smaller and separate reorganizations from changes. Sorry about that.

Reply:
no worries. It sure would be nice if there is a standard way for us to integrate unit tests here ...